src/test/java/seedu/weme/logic/commands/MemeEditCommandTest.java

```diff
@@ -80,7 +82,8 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
 
         String expectedMessage = String.format(MemeEditCommand.MESSAGE_EDIT_MEME_SUCCESS, editedMeme);
 
-        Model expectedModel = new ModelManager(new MemeBook(model.getMemeBook()), new UserPrefs());
+        Model expectedModel = new ModelManager(new MemeBook(model.getMemeBook()), new UserPrefs(),
+                model.getStatsEngine());
```

Comment:
Should construct a new `StatsEngine`, applies to all similar tests.

Reply:
If it is already constructed in the model field declaration, is there a need to make a new one instead of getting from the model field? First argument is using model.getMemeBook() so I thought it should be the same for statsEngine.

Reply:
first argument is constructing a new memebook that holds the same content as model.getMemeBook